scapy/utils.py

```diff
@@ -33,6 +33,14 @@
 ###########
 
 def get_temp_file(keep=False, autoext=""):
+    """Return file path of a temp file.
+
+    params:
+    - keep: if True, the file won't be deleted on closing. False by default
+    - autoext: specify the file extension. Empty by default
+    """
+    if autoext:
```

Comment:
This would change the API.

Reply:
It will, but the `autoext` makes no sense otherwise.
Also, this function was not documented, so shouldn't have been use outside scapy

Reply:
It is not a private function (otherwise, it should start with a leading `_`) so external Scapy modules could legitimately use it. Also, it does mean something: it is actually a suffix (should probably be named `suffix` rather than `autoext`) and the meaning is the same than `NamedTemporaryFile()` `suffix` parameter (see #863).

Reply:
All right